js/filters.js

```diff
@@ -115,9 +145,27 @@ function TypeaheadFilter(elm) {
   var key = this.$body.data('dataset');
   var dataset = typeahead.datasets[key];
   this.typeaheadFilter = new typeaheadFilter.TypeaheadFilter(this.$body, dataset);
+  this.typeaheadFilter.$body.on('change', 'input[type="checkbox"]', this.handleNestedChange.bind(this));
 }
 
 TypeaheadFilter.prototype = Object.create(Filter.prototype);
 TypeaheadFilter.constructor = TypeaheadFilter;
 
+TypeaheadFilter.prototype.handleChange = function() {};
+
+TypeaheadFilter.prototype.handleNestedChange = function(e) {
```

Comment:
Just curious, why is this in here as opposed to typeahead-filter.js (or vice versa, why are the other methods defined in typeahead-filter.js)?